.travis.yml

```diff
@@ -8,6 +8,8 @@ matrix:
     env: GO111MODULE=on
   - go: 1.12.x
     env: GO111MODULE=on
+  - go: 1.13.x
+    env: GO111MODULE=on
```

Comment:
We don't need to set `GO111MODULE` for `1.13`. 

> The GO111MODULE environment variable continues to default to auto, but the auto setting now activates the module-aware mode of the go command whenever the current working directory contains, or is below a directory containing, a go.mod file — even if the current directory is within GOPATH/src. This change simplifies the migration of existing code within GOPATH/src and the ongoing maintenance of module-aware packages alongside non-module-aware importers.

https://golang.org/doc/go1.13#modules

Reply:
@appleboy thanks! fixed.